doc/GuideForCommitters.md

```diff
@@ -167,9 +167,9 @@ possible between them.
     necessary to wait for this build to finish because an equivalent
     of the OMR Acceptance Build was already tested in Step 1.
 
-7.  Verify that the `master` branch of the `eclipse-openj9/openj9-omr` repo
-    contains the Eclipse OMR commit you merged in Step 3.  Make note
-    of the commit SHA for Step 8.
+7.  Verify that the [`master` branch](https://github.com/eclipse-openj9/openj9-omr/commits/master)
```

Comment:
It would look better if master isn't in back quotes. Looking at the formatted output, as it is, each shows seems a different link.

Reply:
Fixed in forced push.  Thanks.